src/goes/goes_converter.py

```diff
@@ -72,27 +81,44 @@ def _check_arguments(self):
         if len(self._input_file_paths) != 16:
             print("ERROR: input_file_paths must contain 16 GOES-16 or GOES-17 data files. One for each ABI channel.")
             good_args = False
+        good_resolutions = [2, 4, 8, 16, 32, 64]
+        if self._resolution not in good_resolutions:
+            print("ERROR: resolution (in km) must be one of the following values: 2 (default), 4, 8, 16, 32, 64.")
+            good_args = False
         if not good_args:
             sys.exit(2)
 
-    def _create_input_data_file_dicts(self):
+    def _initialize(self):
         """
         Create two local dictionaries contained the Goes class instances for brightness temperature (ABI channels 1-6)
```

Comment:
The comment here is mistaken.  Channels 1-6 are reflectances whereas channels 7-16 are the infrared brightness temperatures.

Reply:
This has been resolved.